docs/resources/bgp_peer.md

```diff
@@ -9,7 +9,7 @@ resource "mikrotik_bgp_peer" "peer" {
   name = "bgp-peer-name"
   remote_as = 65533
   remote_address = "172.21.16.20"
-  peer = "name of bgp peer" # mikrotik_bgp_peer.name
+  instance = "name of bgp instance" # mikrotik_bgp_instance.name
```

Comment:
Here is what I was thinking of. Except I would expand on the `mikrotik_bpg_instance` arguments

```
resource "mikrotik_bpg_instance" "instance" {
  [...]
}

resource "mikrotik_bgp_peer" "peer" {
  name = "bgp-peer-name"
  remote_as = 65533
  remote_address = "172.21.16.20"
  instance = mikrotik_bgp_instance.instance.name
}
```